### PR TITLE
G Suite: Update Upsell page for Google Workspace

### PR DESCRIPTION
**Pull request by stephanethomas**
This pull request updates the `Upsell` page for Google Workspace, and makes some minor visual adjustments:

Before | After
------ | -----
screenshot  | screenshot

#### Testing instructions

1. Run `git checkout update/upsell-for-google-workspace` and start your server, or open a [live branch](https://calypso.live/?branch=update/upsell-for-google-workspace)
2. Open the [`Domain Search` page](http://calypso.localhost:3000/domains/add)
3. Select any domain
4. Assert that the `Upsell` page matches the screenshot above
5. Assert that the page looks good when reducing the width of the browser

### CODE REVIEW

File: client/components/gsuite/gsuite-price/index.jsx [hunk 1]
```diff
@@ -54,7 +54,7 @@ const GSuitePrice = ( { currencyCode, product } ) => {
 	return (
 		<div className="gsuite-price">
 			<h4 className="gsuite-price__monthly-price">
-				{ translate( '{{strong}}%(price)s{{/strong}} per user/month', {
+				{ translate( '{{strong}}%(price)s{{/strong}} /user /month', {
```
Comment (stephanethomas) [inline client/components/gsuite/gsuite-price/index.jsx:57]: I've updated this string to follow the same format than on the `Email Comparison` page.

File: client/components/upgrades/gsuite/gsuite-upsell-card/index.jsx [hunk 2]
```diff
@@ -85,10 +86,12 @@ const GSuiteUpsellCard = ( {
 			<CompactCard>
 				<header className="gsuite-upsell-card__header">
 					<h2 className="gsuite-upsell-card__title">
-						{ translate( 'Add professional email from G Suite by Google Cloud to %(domain)s', {
+						{ translate( 'Add professional email from %(productFamily)s to %(domain)s', {
 							args: {
 								domain,
+								productFamily: getGoogleMailServiceFamily( productSlug ),
```
Comment (stephanethomas) [inline client/components/upgrades/gsuite/gsuite-upsell-card/index.jsx:92]: The `productSlug` parameter will be correctly handled on https://github.com/Automattic/wp-calypso/pull/48761 has been merged.

File: client/components/gsuite/gsuite-price/index.jsx (hunk 1 above)
Comment (a8ci18n) [inline client/components/gsuite/gsuite-price/index.jsx:57]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **24** times:
`translate( '{{strong}}%(price)s{{/strong}} per user / month' )` *ES Score: 7*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/upsell-for-google-workspace)

**Review by olaseni — APPROVED** [review]
Looks good.

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/5475679

Thank you @stephanethomas for including a screenshot in the description! This is really helpful for our translators.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/5475679).

**Pull request merged**